### ISSUE TITLE
EKS Launch Template now has multiple mentions of "NODE_TAINT"

### ISSUE THREAD

**Issue by syvanen** [awslabs/fleetiq-adapter-for-agones#11]
The `awk` on the [quickinstall.sh line 161 ](https://github.com/awslabs/fleetiq-adapter-for-agones/blob/54a4267cabd7cebd7c6784ceecbc0d93b5986a19/Agones_EKS_FleetIQ_Integration_Package%5BBETA%5D/quick_install/fleet_eks_agones_quickinstall.sh#L161)fails because there's multiple lines with `NODE_TAINTS=`in the launch template now. 

By adding `-m1` limits the match to first hit making the `awk` replacement work again. Without this the daemon will fail to add/update the taint in [main.py line 206 ](https://github.com/awslabs/fleetiq-adapter-for-agones/blob/54a4267cabd7cebd7c6784ceecbc0d93b5986a19/gamelift-daemon/src/main.py#L206) to fail as there are no taints defined on the node.

Error on logs:
```
Registering game server
The game server is already registered
The instance is HEALTHY
Updating game server health
Claiming game server
The instance has already been claimed
Changing status to utilized
Traceback (most recent call last):
File "main.py", line 347, in <module>
main()
File "/usr/local/lib/python3.8/site-packages/click/core.py", line 829, in __call__
return self.main(*args, **kwargs)
File "/usr/local/lib/python3.8/site-packages/click/core.py", line 782, in main
rv = self.invoke(ctx)
File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
return ctx.invoke(self.callback, **ctx.params)
File "/usr/local/lib/python3.8/site-packages/click/core.py", line 610, in invoke
return callback(*args, **kwargs)
File "main.py", line 322, in main
initialize_game_server(GameServerGroupName=game_server_group_name, GameServerId=game_server_id, InstanceId=instance_id)
File "main.py", line 207, in initialize_game_server
taints.append(taint)
AttributeError: 'NoneType' object has no attribute 'append'
```

**Comment by jicowan** [comment]
Thanks @syvanen. 
cc @trevorrobertsjr

**Comment by trevorrobertsjr** [comment]
Fixed by #12
